src/browser/navigation-panel.js

```diff
@@ -94,7 +111,7 @@ const NavigationPanel = Component(({key, input, tabStrip, webViewer, title}) =>
                         + (webViewer.get('securityState') == 'secure' ? ' ssl' : '')
                         + (webViewer.get('securityExtendedValidation') ? ' sslv' : '')
   }, [
-    WindowControls({key: 'controls'}),
+    WindowControls({key: 'controls', webViewer}),
```

Comment:
I think it would be better to do `location: webViewer.get('location')` instead to avoid generating `WindowControls` even if nothing about the location changes.